### PR TITLE
search feature functioning. need to double check too many api calls

### PR DESCRIPTION
**Pull request by l1llyp**

### CODE REVIEW

**Comment by ghost** [comment]
pair programming to fix search feature

File: boiler/src/client/components/Search.jsx [hunk 1]
```diff
@@ -1,32 +1,65 @@
...
-    return axios.get('https://cors-anywhere.herokuapp.com/https://api.yelp.com/v3/businesses/search?location=neworleans&categories=crawfish', {
+  // const [ store, useStore ] = useState([]);
+  
+  const [restaurants, setRestaurants] = useState([]);
+  
+  
+  const getRestaurants = () => {
+    axios.get('/api/restaurants')
+    .then((results) => {
+      console.log('results', results.data);
```
Comment (copelka) [inline boiler/src/client/components/Search.jsx:20]: once you get it working, remove console logs before pushing to github (don't clutter your teammates' consoles)

File: boiler/src/client/components/Search.jsx [hunk 2]
```diff
@@ -1,32 +1,65 @@
...
+      setRestaurants(results.data);
```
Comment (copelka) [inline boiler/src/client/components/Search.jsx:21]: destructure

File: boiler/src/client/components/Search.jsx [hunk 3]
```diff
@@ -1,32 +1,65 @@
...
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  };
+  
+  const getCrawfish = (location) => {
+    console.log('this is the location', location);
```
Comment (copelka) [inline boiler/src/client/components/Search.jsx:29]: remove console logs

File: boiler/src/client/components/Search.jsx [hunk 4]
```diff
@@ -1,32 +1,65 @@
...
-      .then((response) => response.data.businesses)
-      .then((businesses) => { useStore(businesses)
-        businesses.forEach((store) => {
-          axios.post('/api/restaurants', {title: store.name, price: store.price, address: store.location.address1, lat: store.coordinates.latitude, long: store.coordinates.longitude, imageUrl: store.image_url})
-            .then((res) => { console.log('Saved Restaurant' ); })
-            .catch((err) => { console.log('Unable to save restaurant'); });
-        });
-      })
-      .catch(err => console.error('error in yelp api call: ', err));
+    .then((response) => response.data.businesses)
```
Comment (copelka) [inline boiler/src/client/components/Search.jsx:37]: destructure & combine with next promise

**Pull request merged**